jetty-ee10/jetty-ee10-proxy/src/main/config/modules/ee10-proxy.mod

```diff
@@ -18,10 +17,49 @@ lib/jetty-ee10-proxy-${jetty.version}.jar
 etc/jetty-ee10-proxy.xml
 
 [ini-template]
-## Proxy Configuration
-# jetty.proxy.servletClass=org.eclipse.jetty.ee10.proxy.ProxyServlet
+# tag::documentation[]
+## The proxy contextPath.
+# jetty.proxy.contextPath=/
+
+## The proxy Servlet class.
+# jetty.proxy.servletClass=org.eclipse.jetty.ee10.proxy.ProxyServlet$Transparent
+
+## The proxy Servlet mapping.
 # jetty.proxy.servletMapping=/*
+
+## The target URI to proxy to, for example http://backend:9090/path.
+## This parameter is mandatory.
+# jetty.proxy.proxyTo=
+
+## The prefix of the URI path in context to strip from the incoming URI.
+## For example, incoming URI http://proxy:8080/ctx/foo/bar, with
+## jetty.proxy.prefix=/foo becomes http://backend:9090/path/bar.
+# jetty.proxy.prefix=
+
+## The proxy HttpClient max number of threads.
 # jetty.proxy.maxThreads=128
+
+## The proxy HttpClient max number of connections.
 # jetty.proxy.maxConnections=256
+
+## The proxy HttpClient idle timeout.
 # jetty.proxy.idleTimeout=30000
+
+## The proxy HttpClient total request timeout.
 # jetty.proxy.timeout=60000
```

Comment:
Is it possible to have a configuration mode where it just uses the servers ThreadPool rather than its own?